packages/vite-plugin-kit-routes/src/test/ROUTES_format-route-underscore.ts

```diff
@@ -0,0 +1,174 @@
+ * 
+ *      >> DO NOT EDIT THIS FILE MANUALLY <<
+ */
+
+/**
+ * PAGES
+ */
+const PAGES = {
+  "_ROOT": `/`,
+  "subGroup": `/subGroup`,
+  "subGroup2": (params: { first: (string | number) }) => {
+    return `/subGroup2${appendSp({ first: params?.first })}`
+  },
+  "contract": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }, sp?: Record<string, string | number>) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/contract${appendSp(sp)}`
+  },
+  "contract_id": (params: { id: (string | number), lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/contract/${params.id}`
+  },
+  "gp_one": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/gp/one`
+  },
+  "gp_two": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/gp/two`
+  },
+  "main": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/main`
+  },
+  "match_id_int": (params: { id: (string | number), lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/match/${params.id}`
+  },
+  "site": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string), limit?: (number) }, sp?: Record<string, string | number>) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/site${appendSp({ limit: params?.limit, ...sp })}`
+  },
+  "site_id": (params?: { lang?: ('fr' | 'hu' | undefined), id?: (string), limit?: (number), demo?: (string) }) => {
+    params = params ?? {}
+    params.lang = params.lang ?? "fr"; 
+    params.id = params.id ?? "Vienna"; 
+    return `${params?.lang ? `/${params?.lang}`: ''}/site/${params.id}${appendSp({ limit: params?.limit, demo: params?.demo })}`
+  },
+  "site_contract_siteId_contractId": (params: { siteId: (string | number), contractId: (string | number), lang?: ('fr' | 'en' | 'hu' | 'at' | string), limit?: (number) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/site_contract/${params.siteId}-${params.contractId}${appendSp({ limit: params?.limit })}`
+  },
+  "a_rest_z": (params: { rest: (string | number)[] }) => {
+    return `/a/${params.rest?.join('/')}/z`
+  },
+  "lay_normal": `/lay/normal`,
+  "lay_root_layout": `/lay/root-layout`,
+  "lay_skip": `/lay/skip`
+}
+
+/**
+ * SERVERS
+ */
+const SERVERS = {
+  "GET contract": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/contract`
+  },
+  "POST contract": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/contract`
+  },
+  "GET site": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/site`
+  },
+  "GET api_graphql": `/api/graphql`,
+  "POST api_graphql": `/api/graphql`
```

Comment:
Same principle, but in _ mode

Reply:
Here you are in the `route(symbol)` test.
`route('GET site')`

for `object(symbol)` you are right it will
`SERVER.GET_contrat`

I'm about to push something

Reply:
Oh! Got you. Sorry I'm getting very confused with all the formats possibilities that the plugin now offers.

Giving users the choice is good, but being a little opinionated is good as well, you don't want to spread too thin.